app/controllers/errors_controller.rb

```diff
@@ -11,15 +11,6 @@ def show
 
   private
 
-  def select_foodcoop
-    foodcoop = params[:foodcoop]
-    if FoodsoftConfig.allowed_foodcoop? foodcoop
-      FoodsoftConfig.select_foodcoop foodcoop
-    else
-      FoodsoftConfig.select_default_foodcoop
-    end
-  end
-
```

Comment:
This is already provided by the `ApplicationController`, and that handles single coop installs better.

Reply:
i'd create a separate commit for that, since it's unrelated

Reply:
Ah, thanks - I intended to but then forgot!